notification-portlet-webapp/src/main/webapp/scripts/jquery.notice.js

```diff
@@ -38,7 +38,8 @@ if (!upnotice.init) {
         link:            '.link',
         actions:         '.notification-actions',
         actionTemplate:  '.action-template',
-        summaryTemplate: '.summary-template'
+        summaryTemplate: '.summary-template',
+        completedBadge:  '.completed-badge'
```

Comment:
The name implies the state is set to completed which is not necessarily the case since it is configurable.